packages/itwin/breakdown-trees/src/assets/ClassificationSystems.json

```diff
@@ -1,5 +1,5 @@
 {
-  "$schema": "../../node_modules/@bentley/presentation-common/Ruleset.schema.json",
+  "$schema": "../../node_modules/@itwin/presentation-common/Ruleset.schema.json",
   "id": "ui-framework/ClassificationSystems",
```

Comment:
could you update all these ruleset ids, so theyre prefixed with itwin-breakdown-trees instead?